main.go

```diff
@@ -319,6 +368,21 @@ func main() {
 		log.Println("could not add custom rules chain: ", err)
 	}
 
+	//Custom nodes are static, no need to retrieve them from gossip
+	for _, node := range cfg.CustomNodes {
+		log.Println("Adding custom node: ", node.Name, " with IP: ", node.Ip, " to set: ", cfg.CustomNodeClass.Name)
+		err := ipset.Add(gossipSet, node.Ip)
+
+		if err != nil {
+			log.Println("Failed to add node to gossip set: ", err)
+		}
```

Comment:
Maybe don't add these to the gossip set and only to the custom node class name? That allows more flexibility for the operator running this on their validator.

Reply:
We went back and forth on this, but some people are *only* using a binary gossip/non-gossip check and this was the easiest way to support them.